### ISSUE TITLE
Domains being blocked trying to read from non-existent file

### ISSUE THREAD

**Issue by PromoFaux** [jacobsalmela/AdminLTE#11]
https://github.com/jacobsalmela/pi-hole/issues/191#issuecomment-172497941

See issue on the main pi-hole repo for details.

**Comment by PromoFaux** [comment]
Ah wait. I see this was changed in #8, but <code>api.php</code> was not updated at the same time!

in which case I'll make another commit to fix it to the correct <code>eventhorizon.txt</code>

**Comment by jacobsalmela** [comment]
Yeah, as in #8, it would show double the amount of domains blocked if IPv6 was enabled.

**Comment by PromoFaux** [comment]
fixed with #15 and [this pull](https://github.com/jacobsalmela/pi-hole/pull/193)

**Comment by jacobsalmela** [comment]
You have some mad skills.  So after I merge the new `chronometer.sh`, I should merge #15 here to fix this issue?

**Comment by PromoFaux** [comment]
Yep. Have tested locally, you might want to just pull over the 3 files to your local pi and sanity check first. Tried to pull into a new branch on the main repo, but could not see an option to.

**Comment by jacobsalmela** [comment]
I just tried your `chronometer.sh` and got this:

```
pi@raspberrypi ~ $ sudo /usr/local/bin/chronometer.sh 
/usr/local/bin/chronometer.sh: line 132: unexpected EOF while looking for matching `"'
/usr/local/bin/chronometer.sh: line 136: syntax error: unexpected end of file
```

**Comment by jacobsalmela** [comment]
Just found it: a missing quotation on [line 115](https://github.com/PromoFaux/pi-hole/blob/Chronometer/advanced/Scripts/chronometer.sh#L115)

**Comment by PromoFaux** [comment]
Odd. Fixed.

**Comment by jacobsalmela** [comment]
Fixed by #15
